template.json

```diff
@@ -321,37 +394,51 @@
     "archetype" : {
       "templates" : ["keywords", "XP"],
       "tier" : 1,
-      "species" : "",
       "attributes" : [],
       "skills" : [],
       "ability" : "",
       "wargear" : "",
       "influence" : 0
     },
     "augmetics" : {
-      "templates" : ["effect", "value", "rarity", "keywords"]
+      "templates" : ["value", "rarity", "keywords", "effect"]
     },
     "equipment" : {
-      "templates" : ["effect", "value", "rarity", "keywords"]
+      "templates" : ["value", "rarity", "keywords", "effect", "quantity"]
+    },
+    "injury" : {
+      "templates" : ["effect"],
+      "severity" : ""
+    },
+    "misc" : {
+      "templates" : ["quantity"]
     },
     "mutation" : {
       "templates" : ["effect"]
     },
     "power" : {
-      "templates" : ["XP", "keywords"],
+      "templates" : ["XP", "keywords", "effect"],
```

Comment:
Can description suffice for "effect"? All items have descriptions. I would like to reserve effect to have some programmability. 

**For Example**
An augmetic item could have an `item.data.effect.bonus.attribute.str : 1` that can be applied on the fly
(This could not work or not make sense so open to input)

Reply:
It's a good distinction.  However, I do think there should probably be a difference drawn between the fluffy lore description and the meat and potato ability/effect itself, so it isn't a big block of text needing to be scrolled through.

For example, time for some fun inconsistency, there are some talents for instance that provide + Double Rank Bonus to Passive Awareness, so they are definitely programmable. item.data.effect.bonus.mental.passaware: 3

Then there's powers of course that give you vague role playing abilities, such as Psychometry, that are really just a block of text left to be interpreted by the GM/Player.

Is there a layering that can be achieved potentially?

1. Description = Fluff
2. Ability/Effect = Actual Important Ability Info
3. Effect/Mod = Programmable effect